addon/helpers/list.js

```diff
@@ -2,6 +2,16 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([array]) {
+    if (array === undefined) {
+      return [];
+    } else if (array && array.length != null && array.forEach) {
```

Comment:
This check for `forEach` makes a lot of sense, but I'm wondering if this arm of the `if` shouldn't just be an `instanceof Array` check.

If we want to do duck typing, we can do it in another `else if` and then copy it so that the list microstate always has an instance of `Array` to work with.